access/services/project.go

```diff
@@ -0,0 +1,165 @@
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/jfrog/jfrog-client-go/auth"
+	"github.com/jfrog/jfrog-client-go/http/jfroghttpclient"
+	clientutils "github.com/jfrog/jfrog-client-go/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/httputils"
+)
+
+const projectsApi = "v1/projects"
+
+type ProjectParams struct {
+	ProjectDetails Project
+}
+
+func NewProjectParams() ProjectParams {
+	return ProjectParams{}
+}
+
+type Project struct {
+	DisplayName       string           `json:"display_name,omitempty"`
+	Description       string           `json:"description,omitempty"`
+	AdminPrivileges   *AdminPrivileges `json:"admin_privileges,omitempty"`
+	SoftLimit         bool             `json:"soft_limit,omitempty"`
+	StorageQuotaBytes float64          `json:"storage_quota_bytes,omitempty"`
+	ProjectKey        string           `json:"project_key,omitempty"`
+}
+
+type AdminPrivileges struct {
+	ManageMembers   bool `json:"manage_members,omitempty"`
+	ManageResources bool `json:"manage_resources,omitempty"`
+	IndexResources  bool `json:"index_resources,omitempty"`
+}
+
+type ProjectService struct {
+	client         *jfroghttpclient.JfrogHttpClient
+	ServiceDetails auth.ServiceDetails
+}
+
+func NewProjectService(client *jfroghttpclient.JfrogHttpClient) *ProjectService {
+	return &ProjectService{client: client}
+}
+
+func (us *ProjectService) SetArtifactoryDetails(rt auth.ServiceDetails) {
```

Comment:
This method seem to be unused and should probably be removed. Also, it's name is probably wrong, because this an Access service and not an Artifactory service.